app/Lib/Export/HidsBroExport.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+App::uses('HidsExport', 'Export');
+
+/**
+ * Created by IntelliJ IDEA.
+ * User: ppanero
+ * Date: 11/08/16
+ * Time: 11:04
+ */
+class HidsBroExport extends HidsExport
+{
+
+
+// below overwrite functions from HidsExport
+	public function export($items, $type = 'MD5', $continue = false)
+	{
+
+		$orgsName = array();
+		if (!empty($items)) {
+			foreach ($items as &$item) {
```

Comment:
is it really necessary to use the &$item here instead of simple $item  ? (same in line 37)